src/storage/db_interface_common.py

```diff
@@ -167,20 +165,20 @@ def retrieve_analysis(self, sanitized_dict, analysis_filter=None):
         for key in analysis_filter:
             try:
                 if sanitized_dict[key]['file_system_flag']:
-                    logging.debug('Retrieving stored file {}'.format(key))
+                    logging.debug(f'Retrieving stored file {key}')
                     sanitized_dict[key].pop('file_system_flag')
                     sanitized_dict[key] = self._retrieve_binaries(sanitized_dict, key)
                 else:
                     sanitized_dict[key].pop('file_system_flag')
             except (KeyError, IndexError, AttributeError, TypeError, pickle.PickleError) as error:
-                logging.debug('Could not retrieve information: {} {}'.format(type(error), error))
+                logging.debug(f'Could not retrieve information: {type(error)} {error}')
```

Comment:
While we are at it, we should probably change this to `error()` or at least `warning()` and add `exc_info=True` (and maybe remove one of the `{error}`s or both)